pandas/tests/indexing/multiindex/test_loc.py

```diff
@@ -412,13 +409,6 @@ def test_loc_getitem_duplicates_multiindex_missing_indexers(indexer, pos):
         tm.assert_series_equal(result, expected)
 
 
-def test_series_loc_getitem_fancy(multiindex_year_month_day_dataframe_random_data):
```

Comment:
this is duplicated somewhere?

Reply:
Yes, was tricky to find.

test_loc_getitem_list_of_tuples_with_multiindex same file line 81. Different names but content is the same

Reply:
nice.  these can be tough, but always fulfilling to rip out

Reply:
Yes definitely :)